undertow/WFLY-17766_AccessLog_Termination.adoc

```diff
@@ -0,0 +1,114 @@
+= [Preview] Add ability to configure access log termination schedule and delay
+:author:            Bartosz Baranowski
+:email:             bbaranow@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This RFE(https://issues.redhat.com/browse/WFLY-17766) is part of bug fix for https://issues.redhat.com/browse/WFLY-13933 / https://issues.redhat.com/browse/JBEAP-20056 . In order to avoid bad user experience and possible follow up, it would be
```

Comment:
Minor: for readability (since these are published to wildfly docs) it would be good to add the issue links as hyperlinks.

Reply:
Please also add the stability level to the PR title.